src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (x) => {
```

Comment:
The parameter 'x' should be checked to ensure it is a number before adding it to 'sum'. This will prevent potential bugs if the function is used incorrectly with non-numeric values.